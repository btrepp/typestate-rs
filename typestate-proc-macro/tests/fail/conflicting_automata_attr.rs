use typestate_proc_macro::typestate;

#[typestate]
mod m {
    #[state]
    #[automata] // this should error because it is the second one
    struct S {}
}

fn main() {}