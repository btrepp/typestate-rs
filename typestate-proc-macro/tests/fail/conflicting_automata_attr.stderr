error: Conflicting attributes are declared.
 --> $DIR/conflicting_automata_attr.rs:6:5
  |
6 |     #[automaton] // this should error because it is the second one
  |     ^^^^^^^^^^^^
